### PR TITLE
Update WatchGuardFirebox.txt

### PR DESCRIPTION
**Pull request by ecosystempo**
Try to update

WatchGuard syslog Parser

### CODE REVIEW

File: Parsers/WatchGuardFirebox.txt [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+};
 Syslog
-| extend Interface = extract("msg_id=\"3100-002C\" \\[(.*)\\]",1,SyslogMessage),
-Intstatus = extract("Interface link status changed to ([\\w\\s]+)",1,SyslogMessage),
+| extend PolicyName = replace("-00","",extract(@"\s\(([-\w\s]*?(-00|\sPolicy|DVCP-BOVPN-Allow-in))\)$",1,SyslogMessage, typeof(string))),
+PolicyAction = extract("msg_id=\".*?\"\\s(\\w+?)\\s.*(Policy|-00|DVCP-BOVPN-Allow-in)\\)$",1,SyslogMessage),
+ProxyName = extract("Proxy.*?: ([\\w\\s]+)",1,SyslogMessage),
+Application = extract("app_name=\"(.*?)\"",1,SyslogMessage),
+MessageId = extract("msg_id=\"(.*?)\"",1,SyslogMessage),
+Message = extract("msg=\"(.*?)\"",1,SyslogMessage),
```
Comment (v-admahe) [inline Parsers/WatchGuardFirebox.txt:28]: Should it not be 'EventMessage'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

File: Parsers/WatchGuardFirebox.txt [hunk 2]
```diff
@@ -1,25 +1,43 @@
...
-Application = extract("app_name=\"(.*?)\"",1,SyslogMessage),
-Proxy = extract("Proxy.*: ([\\w\\s]+)",1,SyslogMessage),
-Action = extract("msg_id=\".*?\" (\\w+?) ",1,SyslogMessage)
+GeoDst = extract("geo_dst=\"(.*?)\"",1,SyslogMessage),
+GeoSrc = extract("geo_src=\"(.*?)\"",1,SyslogMessage),
+SentBytes = extract("sent_bytes=\"(.*?)\"",1,SyslogMessage),
+RcvdBytes = extract("rcvd_bytes=\"(.*?)\"",1,SyslogMessage),
+FireboxVersion = extract("Watchguard loggerd (.*?) ",1,SyslogMessage),
+FireboxManageUser = extract("Management user (.*?)@",1,SyslogMessage),
+SourceIP = fromat_result(extract_all(@"(:\s(?P<srcIp1>(\d{1,3}\.){3}\d{1,3}):\d{1,5}\s->\s)|(\s(?P<srcIp2>(\d{1,3}\.){3}\d{1,3})\s(\d{1,3}\.){3}\d{1,3}\s)", dynamic(['srcIp1', 'srcIp2']), SyslogMessage)),
```
Comment (v-admahe) [inline Parsers/WatchGuardFirebox.txt:40]: Should it not be 'SrcIpAddr'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

File: Parsers/WatchGuardFirebox.txt [hunk 3]
```diff
@@ -1,25 +1,43 @@
...
+DestinationIP = fromat_result(extract_all(@"(:\d{1,5}\s->\s(?P<destIp1>(\d{1,3}\.){3}\d{1,3}):\d{1,5}\s)|(\s(\d{1,3}\.){3}\d{1,3}\s(?P<destIp2>(\d{1,3}\.){3}\d{1,3})\s)", dynamic(['destIp1', 'destIp2']), SyslogMessage)),
```
Comment (v-admahe) [inline Parsers/WatchGuardFirebox.txt:41]: Should it not be 'DstIpAddr'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

File: Parsers/WatchGuardFirebox.txt [hunk 4]
```diff
@@ -1,25 +1,43 @@
...
+SourcePort = fromat_result(extract_all(@"(:\s(\d{1,3}\.){3}\d{1,3}:(?P<srcPort1>\d{1,5})\s->\s)|(\s(\d{1,3}\.){3}\d{1,3}\s(\d{1,3}\.){3}\d{1,3}\s(?P<srcPort2>\d{1,5})\s)", dynamic(['srcPort1', 'srcPort2']), SyslogMessage)),
```
Comment (v-admahe) [inline Parsers/WatchGuardFirebox.txt:42]: Should it not be 'SrcPortNumber'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

File: Parsers/WatchGuardFirebox.txt [hunk 5]
```diff
@@ -1,25 +1,43 @@
...
+DestinationPort = fromat_result(extract_all(@"(:\d{1,5}\s->\s(\d{1,3}\.){3}\d{1,3}:(?P<destPort1>\d{1,5})\s)|(\s(\d{1,3}\.){3}\d{1,3}\s(\d{1,3}\.){3}\d{1,3}\s\d{1,5}\s(?P<destPort2>\d{1,5})\s)", dynamic(['destPort1', 'destPort2']), SyslogMessage))
```
Comment (v-admahe) [inline Parsers/WatchGuardFirebox.txt:43]: Should it not be 'DstPortNumber'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

**Review by v-admahe — CHANGES_REQUESTED** [review]

**Comment by ecosystempo** [comment]
Modify done for the change request 

image

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**